src/WebJobs.Script.WebHost/Diagnostics/DeferredLogEntry.cs

```diff
@@ -17,5 +17,7 @@ public class DeferredLogEntry
         public Exception Exception { get; set; }
 
         public EventId EventId { get; set; }
+
+        public string Scope { get; set; }
```

Comment:
Lets worry about adding scope to this type in a separate PR.

Reply:
Added scope provider